src/applications/vaos/project-cheetah/components/ReviewPage.jsx

```diff
@@ -42,24 +42,20 @@ function ReviewPage({
     <div>
       <h1>{pageTitle}</h1>
       <p className="vads-u-margin-top--1 vads-u-margin-bottom--4">
-        Please review the information before confirming your appointments.
+        Make sure everything is correct before submitting your request.
```

Comment:
Is this potentially confusing @laurenernest, saying that you're submitting a request?